content/docs/tasks/traffic-management/egress-gateway.md

```diff
@@ -0,0 +1,316 @@
+---
+title: Configure an Egress Gateway
+description: Describes how to configure Istio to direct traffic to external services through a dedicated gateway service
+weight: 43
+---
+
+> This task uses the new [v1alpha3 traffic management API](/blog/2018/v1alpha3-routing/). The old API has been deprecated and will be removed in the next Istio release. If you need to use the old version, follow the docs [here](https://archive.istio.io/v0.7/docs/tasks/traffic-management/). Note that this task introduces a new concept, namely Egress Gateway, that was not present in previous Istio versions.
+
+The [Control Egress Traffic](/docs/tasks/traffic-management/egress/) task demonstrates how external (outside the Kubernetes cluster) HTTP and HTTPS services can be accessed from applications inside the mesh. A quick reminder: by default, Istio-enabled applications are unable to access URLs outside the cluster. To enable such access, a [ServiceEntry](/docs/reference/config/istio.networking.v1alpha3/#ServiceEntry) for the external service must be defined, or, alternatively, [direct access to external services](/docs/tasks/traffic-management/egress/#calling-external-services-directly) must be configured.
+
+The [TLS Origination for Egress Traffic](/docs/tasks/traffic-management/egress-tls-origination/) task demonstrates how to allow the applications to send HTTP requests to external servers that require HTTPS.
+
+This task describes how to configure Istio to direct the egress traffic through a dedicated service called _Egress Gateway_. We extend the use case described in the [TLS Origination for Egress Traffic](/docs/tasks/traffic-management/egress-tls-origination/) task.
```

Comment:
I thought I already added a comment about this, but it seems to have disappeared. I suggested changing the second sentence to something like the following:

We'll perform the same functions as described in the [TLS Origination for Egress Traffic](/docs/tasks/traffic-management/egress-tls-origination/) task, only this time we'll do them with the addition of an egress gateway.